lib/ansiblelint/utils.py

```diff
@@ -57,7 +57,7 @@
 from yaml.representer import RepresenterError
 
 from ansiblelint._internal.rules import AnsibleParserErrorRule
-from ansiblelint.constants import CUSTOM_RULESDIR_ENVVAR, DEFAULT_RULESDIR, FileType
+from ansiblelint.constants import CUSTOM_RULESDIR_ENVVAR, DEFAULT_RULESDIR, FileType, options
```

Comment:
This is going to be problematic depending on whether `options` import happens before or after calling `main()` :exploding_head: